pkgs/applications/editors/ne/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, ncurses, texinfo, tetex, perl, ghostscript }:
+
+
+stdenv.mkDerivation {
+  name = "ne-3.0.1";
+  version = "3.0.1";
+  src = fetchurl {
+    url = https://github.com/vigna/ne/archive/3.0.1.tar.gz;
+    md5 = "ec3dc82d10c271e95ecde630538f9f19";
```

Comment:
MD5 is pretty broken these days and I'd avoid it unless there's a pressing reason not to. Stick to SHA256 for most things.